.github/workflows/lint-extras.yaml

```diff
@@ -22,7 +22,7 @@ jobs:
           fetch-depth: 0
 
       - name: Lint Code Base
-        uses: super-linter/super-linter/slim@v6
+        uses: super-linter/super-linter/slim@v6.6.0
```

Comment:
opened tracker for https://github.com/csi-addons/kubernetes-csi-addons/issues/609 to swith to next release.